src/native/corehost/hostmisc/utils.cpp

```diff
@@ -468,8 +463,8 @@ pal::string_t get_download_url(const pal::char_t* framework_name, const pal::cha
     const pal::char_t* arch = get_current_arch_name();
     url.append(_X("&arch="));
     url.append(arch);
-    url.append(_X("&rid=") _STRINGIFY(HOST_RID_PLATFORM) _X("-"));
-    url.append(arch);
+    url.append(_X("&rid="));
+    url.append(get_host_runtime_id());
```

Comment:
I wonder if this should use `get_current_runtime_id` instead. I don't know how useful it is to override rid with ENV (haven't really seen it used anywhere), but if it happens it should probably apply everywhere, otherwise it's really weird.

I guess I'm suggesting we don't have `get_host_runtime_id` and only use `get_current_runtime_id` (maybe renamed to `get_runtime_id`).

What do you think?

Reply:
Yeah, I think that makes sense for consistency.